plugins/topology/src/hooks/useK8sObjectsResponse.ts

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { useKubernetesObjects } from '@backstage/plugin-kubernetes';
+import { K8sResourcesContextData } from '../types/types';
+import { useAllWatchResources } from './useAllWatchResources';
+import { useK8sResourcesClusters } from './useK8sResourcesClusters';
+
+export const useK8sObjectsResponse = (
+  watchedResource: string[],
+): K8sResourcesContextData => {
+  const { entity } = useEntity();
+  const { kubernetesObjects, loading, error } = useKubernetesObjects(entity);
+  const [selectedCluster, setSelectedCluster] = useState<number>(0);
```

Comment:
we are setting the default to `0`. what will happen if there are no clusters?

Reply:
It doesnt matter. This will only be used when there are more than 1 available clusters. And by default we want 1st cluster to be selected.